packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -392,7 +391,7 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
   }
 
   private _onRenderSubMenu(subMenuProps: IContextualMenuProps) {
-    return <ContextualMenu {...subMenuProps} />;
+    return <ContextualMenuBase {...subMenuProps} />;
```

Comment:
I don't think this is correct. We will lose all styling if `ContextualMenuBase` is used.

Reply:
If I understand this correctly, `_onRenderSubMenu` wouldn't be used anyways due to the default props passed to `styled`. In that case, we should look at getting rid of this altogether, or perhaps adding a warning that a default implementation must be provided.

Reply:
If returning `<ContextualMenuBase>` is not desirable in this stub function, I think perhaps we can simply throw an error indicating you need to set `onRenderSubMenu`. Because this stub function can only be called if
* User explicitly sets `onRenderSubMenu` property to `null`. I don't think this happens very often.
* User is implementing a `ContextualMenu` from `ContextualMenuBase`. They will need an compilation/runtime error to know what is required, but there seems nothing I can do to break the build during compiling. So I think throwing an error may do.